CONTRIBUTING.md

```diff
@@ -1,3 +1,92 @@
-# Contributing Guide
-> At the moment, we are not accepting external contributions. Please watch this space to know when we open.
+# Contributing Guide for the 'Run Salesforce Code Analyzer' GitHub Action
+> Currently, we aren't accepting external contributions. To know when we open for contributions, periodically check this guide.
 
+To contribute to the Run Code Analyzer GitHub Action, read this guide to understand its operational governance model. This guide also provides contribution requirements and recommendations. Thanks for your interest and contributions.
+
+# Governance Model
+
+## Salesforce Sponsored
+
+The intent and goal of open sourcing this project is to increase the contributor and user base. Note that Salesforce employees have `admin` rights and are the final arbiters which contributions are accepted.
+
+# Issues, requests & ideas
+
+To submit issues, enhancement requests, and discuss ideas, create an issue on the [Issues page](https://github.com/forcedotcom/run-code-analyzer/issues).
+
+### Bug Reports and Fixes
+-  If you find a bug, search for it in the [Issues](https://github.com/forcedotcom/run-code-analyzer/issues), and if it isn't already tracked,
+   [create a new issue](https://github.com/forcedotcom/run-code-analyzer/issues/new). Fill out the "Bug Report" section of the issue template. Even if an issue is closed, feel free to comment and add details, it will still
+   be reviewed.
+-  Issues that have been confirmed as a bug will be labelled `bug`.
+-  If you'd like to submit a fix for a bug, [create a Pull Request](#creating_a_pull_request) and mention the issue number.
+-  In your bug fix pull request, include tests that isolate the bug and verify that it's fixed.
+
+### New Features
+-  To add new functionality to Run Code Analyzer, describe the problem you want to solve (or the enhancement you identifed) in a [new issue](https://github.com/forcedotcom/run-code-analyzer/issues/new).
```

Comment:
To add new functionality to Run Code Analyzer, describe the problem that you want to solve–or the enhancement that you identifed–in a [new issue]